packages/react/ds/src/radio/radios-group.tsx

```diff
@@ -59,11 +69,11 @@ export const RadiosGroup = ({
                 key={`${value}-${index}`}
```

Comment:
Please use self-closing elements when there are no children:

Nope: `<div className="classname"></div>`
Yep: `<div className="classname"/>`

Reply:
If possible please also avoid using the `index` of an array as key property in an element

Reply:
Fixed. Jusy FYI I kept the index as a key because it is used in conjunction with `value` and the `index` won't change in this particular case since we are not mutating the array